.github/workflows/official_release.yml

```diff
@@ -0,0 +1,65 @@
+
+
+# Beware this is a dangerous script 👻
+name: Make an official Release
+
+# Requires manual trigger
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    name: Test changed-files
+    steps:
+      - uses: trstringer/manual-approval@v1
+        with:
+          secret: ${{ github.TOKEN }}
+          approvers: msaroufim,lxning
+          minimum-approvals: 2
+          issue-title: "Offical Release"
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1
+      - run: conda --version
+      - run: conda install -y conda-build anaconda-client
+      - name: Checkout TorchServe
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          python ts_scripts/install_dependencies.py --environment=dev
+          pip install -e .
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Build PyPI & conda nightly binaries
+        run: python binaries/build.py
+
+      - name: Push conda nightly binaries
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.CONDA_NIGHTLY_TOKEN }}
```

Comment:
Incorrect token. This will push to nightly repo

Reply:
just spotted this, put the right one